mention-bot.js

```diff
@@ -221,6 +227,25 @@ function getSortedOwners(
   return sorted_owners;
 }
 
+function getDefaultOwners(
+  files: Array<FileInfo>,
+  whitelist: Array<WhitelistUser>
+): Array<string> {
+  var owners = [];
+
+  whitelist.forEach(function(user) {
```

Comment:
ohh nevermind about my previous comment. I misread this piece of code. forEach and push is the right way to go here.